pinot-common/src/main/java/org/apache/pinot/common/function/TransformFunctionType.java

```diff
@@ -70,25 +70,27 @@ public enum TransformFunctionType {
 
   // predicate functions
   EQUALS("equals"),
-  NOT_EQUALS("not_equals"),
-  GREATER_THAN("greater_than"),
-  GREATER_THAN_OR_EQUAL("greater_than_or_equal"),
+  NOT_EQUALS("not_equals", "notEquals"),
```

Comment:
Do we need these aliases for auto-generated functions? If so, why skipping `LESS_THAN` and `NOT_IN`?

Reply:
lol good catch. :-)

Reply:
technically we dont need any of these. but we already have ComparisonFunctions in scalar with these names so added `lessThan` but not `notIn` b/c we dont have that

Reply:
This is not the correct way to decide whether to add them though. Eventually we need scalar function for all of them, but that is orthogonal to this problem.
Does postgreSQL support these functions? If so, we need to add them, if not, we should not add them

Reply:
So 2 options. Either we allow underscore on both scalar and transform. Or we don't allow them in neither.

Problem is that transform function in v1 allows underscore so for backward compatibility we need to register them. It is not about syntax. 

For syntax sake we should not allow underscore format all together.

Reply:
discussed offline. will remove these in separate PR as we should not allow these function explicitly being called. They are not consider SqlFunction but SqlOperators to begin with

due to the fact that we do allow these already, we will use a separate PR to deprecate them

Reply:
see #11734